pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -142,6 +144,105 @@ def _save_input_params(self, idata) -> None:
         idata.attrs["validate_data"] = json.dumps(self.validate_data)
         idata.attrs["yearly_seasonality"] = json.dumps(self.yearly_seasonality)
 
+    def _create_likelihood_distribution(
+        self,
+        dist: Dict,
+        mu: SharedVariable,
```

Comment:
Already merged, but this type-hint is wrong: It should be TensorVariable

Reply:
I'll correct the hint on the notebook PR.

Reply:
Added on https://github.com/pymc-labs/pymc-marketing/pull/443